encoding/da.go

```diff
@@ -649,8 +661,10 @@ func GetCodecVersion(config *params.ChainConfig, blockHeight, blockTimestamp uin
 		return CodecV2
 	} else if !config.IsDarwinV2(blockTimestamp) {
 		return CodecV3
-	} else {
+	} else if !config.IsEuclid(blockTimestamp) {
 		return CodecV4
+	} else {
+		return CodecV6
```

Comment:
Adding a comment here why `CodecV6` is "skipped" would be great

Reply:
Do you mean `CodecV5`?